assets/training/finetune_acft_image/environments/acft_image_huggingface/asset.yaml

```diff
@@ -1,5 +1,5 @@
 name: acft-transformers-image-gpu
-version: "3"
+version: "4"
```

Comment:
can `auto` be used?

Reply:
Currently, intentionally keeping version numbering.